### PR TITLE
Show seed admin address

### PR DESCRIPTION
**Pull request by Peartes**
Display the seed admin address instead of the "you" in the stages 7 and 8 of the wizard. This addresses this [issue](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=956b685b35f941f09bf91fcc8d500783)

### CODE REVIEW

File: src/newSeed/baseStage.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export interface IWizardState {
   requiredSeedDeposit?: number;
   requiredSeedFee?: number;
   primeDaoAddress?: Address;
+  seedAdminAddress?: Address;
```
Comment (dkent600) [inline src/newSeed/baseStage.ts:24]: why add this? why not just use `seedConfig.seedDetails.adminAddress`?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/baseStage.ts (hunk 1 above)
Comment (Peartes) [reply src/newSeed/baseStage.ts:24]: Stage8 also displays the seed admin address, but the seedConfig state gets cleared in stage7 upon creating a seed. So, I added that information to the wizard state to allow it to persist until stage8.

File: src/newSeed/stage4.ts [hunk 2]
```diff
@@ -82,6 +82,7 @@ export class Stage4 extends BaseStage {
     temp = this.endDate;
     temp.setHours(Number.parseInt(endTimes[0]), Number.parseInt(endTimes[1]));
     this.seedConfig.seedDetails.endDate = this.dateService.toISOString(this.dateService.translateLocalToUtc(temp));
+    this.wizardState.seedAdminAddress = this.seedConfig.seedDetails.adminAddress;
```
Comment (dkent600) [inline src/newSeed/stage4.ts:85]: can you please add a comment here to explain why this is needed?  Thanks

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage4.ts (hunk 2 above)
Comment (Peartes) [reply src/newSeed/stage4.ts:85]: Okay will do

**Review by dkent600 — APPROVED** [review]

**Pull request merged**